rmw_fastrtps_cpp/README.md

```diff
@@ -1,9 +1,9 @@
 # `rmw_fastrtps_cpp`
 
-`rmw_fastrtps_cpp` implements the ROS middleware interface using eProsima FastRTPS static code generation in C++.
+`rmw_fastrtps_cpp` implements the ROS middleware interface using eProsima Fast DDS static code generation in C++.
 
 For more information see the repository level [README](../README.md)
 
 ## Quality Declaration
 
-This package claims to be in the **Quality Level 4** category, see the [Quality Declaration](QUALITY_DECLARATION.md) for more details.
+This package claims to be in the **Quality Level 2** category, see the [Quality Declaration](QUALITY_DECLARATION.md) for more details.
```

Comment:
There is an inconsistency between this claim (QL2) and the one in the Quality Declaration for `rmw_fastrtps_cpp` package (QL3)